src/pages/core/plugins/freeze-delegate.md

```
@@ -45,6 +52,18 @@ await addPlugin(umi, {
 
 {% /dialect %}
 
+{% dialect title="Rust CPI" id="rust_cpi" %}
+```rust
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)